tools/templates/extras/old_const.jinja2

```diff
@@ -0,0 +1,33 @@
+def const(
```

Comment:
What is the motivation to keep the old version around?

Reply:
I guess I can't come up with one, so we can remove it. It will live forever in the history.